sample/src/main/java/io/getstream/chat/example/MainActivity.java

```diff
@@ -206,7 +207,10 @@ void createNewChannel(String channelName) {
         extraData.put("members", members);
 
         String channelId = channelName.replaceAll(" ", "-").toLowerCase();
+
         Channel channel = new Channel(client, ModelType.channel_messaging, channelId, extraData);
+        Config conf = new Config();
```

Comment:
What is this for?

Reply:
Removed. (i forgot what i planned to do with this)